src/components/Filters/Indicators.js

```diff
@@ -1,8 +1,25 @@
+// @flow
```

Comment:
Note that I didn't need to fix any other file's reference to this one like in Typescript https://github.com/HopkinsIDD/covid-dashboard-app/pull/86/files#diff-e20d0897545286867d50af9026addf07R7

Reply:
That would make migration much more seamless, for sure.